pkg/docker/docker.go

```diff
@@ -287,6 +291,15 @@ func (d *stiDocker) GetImageWorkdir(name string) (string, error) {
 	return workdir, nil
 }
 
+// GetImageEntrypoint returns the ENTRYPOINT property for the given image name.
+func (d *stiDocker) GetImageEntrypoint(name string) ([]string, error) {
+	image, err := d.client.InspectImage(name)
+	if err != nil {
+		return nil, err
```

Comment:
In most of functions we're throwing `errors.NewInspectImageError()` but in some we don't do it. Do you know why? Do we have some rule about it that I don't know?